protobuf/grSim_Commands.proto

```diff
@@ -0,0 +1,29 @@
+message grSim_Robot_Command {
```

Comment:
Can you add 
```
syntax="proto2";
```
to the top of all the protobuf files to avoid warnings on proto3?